server-common/src/main/java/org/apache/kafka/server/config/KraftInternals.java

```diff
@@ -16,12 +16,11 @@
  */
 package org.apache.kafka.server.config;
 
-public class ZooKeeperInternals {
+public class KraftInternals {
```

Comment:
I don't think this name makes sense - there's nothing specific to kraft in this code. Can we move this constant to one of the quota classes and have the others reference it from there?

Reply:
Removing this would affect the default username and default client-id. Since this is a breaking change, I suggest we maintain consistency by keeping these default values. WDYT?

Reply:
@m1a2st Could you please clean up the usage of `ZooKeeperInternals.DEFAULT_STRING` first? We are currently using `ZooKeeperInternals.DEFAULT_STRING` as a specific string for compatibility with the zk handler (QuotaConfigHandler). Since #18617 will remove `QuotaConfigHandler`, the kraft `ClientQuotaMetadataManager` no longer needs to use `ZooKeeperInternals.DEFAULT_STRING`.

After addressing above comment, we can move `<default>` to be a internal constant of `ClientQuotaManager` to ensure the compatibility of metrics `kafka.server:type={Produce|Fetch},user=([-.\w]+),client-id=([-.\w]+)`